lib/msal-browser/docs/caching.md

```diff
@@ -1,3 +1,88 @@
-## Cache Storage
+# Caching
 
-We offer two methods of storage for Msal, `localStorage` and `sessionStorage`.  Our recommendation is to use `sessionStorage` because it is more secure in storing tokens that are acquired by your users, but `localStorage` will give you Single Sign On accross tabs and user sessions.  We encourage you to explore the options and make the best decision for your application.
+When MSAL acquires a token, it caches it for future usage. MSAL manages token lifetimes and refreshing for you. APIs like `acquireTokenSilent()` retrieves tokens from the cache for a given account.
+
+```javascript
+const msal = require('@azure/msal-browser');
+
+// Create the msal application object
+const pca = new msal.PublicClientApplication({
+    auth: {
+        clientId: "Enter_the_Application_Id_Here", // e.g. "b1b60dca-c49d-496e-9851-xxxxxxxxxxxx" (guid)
+        authority: "https://login.microsoftonline.com/Enter_the_Tenant_Info_Here", // e.g. "common" or your tenantId (guid),
+        redirectUri: "/"
+    },
+    cache: {
+       cacheLocation: "sessionStorage"
+    }
+});
+
+/**
+* login* and acquireToken* APIs return an account object containing "homeAccountId"
+* you should keep a record of this in your app and use it later on when calling acquireTokenSilent
```

Comment:
Instead, should mention using `setActiveAccount`.

Reply:
Also, the code shippets below probably aren't needed for this doc.

Reply:
removed the snippet